cmd/root.go

```diff
@@ -59,6 +59,7 @@ func init() {
 
 	rootCmd.AddCommand(initCmd)
 	initCmd.Flags().StringVarP(&initFlagReposFile, "file", "f", "", "get repos from a file instead of searching")
+	initCmd.Flags().StringVarP(&repoSearchQuery, "repo-search", "rs", "", "get repos from a github repo search")
```

Comment:
nit, seems easier to do a bool flag for repo/code since we'll never have both a query and a "repoQuery" simultaneously, and they're essentially identical